### PR TITLE
initial version with newsletters #4 to #8

### PR DESCRIPTION
**Pull request by greenido**
Hey @petele ,

This is the first (small :) step of adding our Newsletter content to updates. I will add all the other 7 after we will talk how/when to do it.

/cc @PaulKinlan

### CODE REVIEW

File: src/content/en/updates/posts/2015/06/edge-of-the-wed-4.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+---
+
+##Edge Of The Web \#4
+
+
+###Chrome 43 (Stable)  
+
+<img src="/web/updates/images/2015-09-27-edge-of-the-wed-7/chrome_256.png">
+
+* The[ ](https://fetch.spec.whatwg.org/)[Fetch
```
Comment (gauntface) [inline src/content/en/updates/posts/2015/06/edge-of-the-wed-4.md]: Formatting is a bit off here.

File: src/content/en/updates/posts/2015/06/edge-of-the-wed-4.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+
+###Chrome 43 (Stable)  
+
+<img src="/web/updates/images/2015-09-27-edge-of-the-wed-7/chrome_256.png">
+
+* The[ ](https://fetch.spec.whatwg.org/)[Fetch 
+  API](https://fetch.spec.whatwg.org/) now allows developers to directly operate 
+  on and incrementally release the bytes of[ 
+  ](https://googlechrome.github.io/samples/fetch-api/fetch-response-stream.html)[streamed
```
Comment (gauntface) [inline src/content/en/updates/posts/2015/06/edge-of-the-wed-4.md]: Same issue as above.

**Comment by gauntface** [comment]
Looks good apart from the empty links that seem to occur in several places throughout the docs.

**Comment by PaulKinlan** [comment]
Just catching up on this, the plan is to post these posts on medium instead to route traffic to the site.

**Comment by gauntface** [comment]
So can this PR be closed?

**Comment by PaulKinlan** [comment]
I think I did.

**Pull request closed without merging**